packages/dapp-high-roller/src/components/app-provider/app-provider.tsx

```diff
@@ -0,0 +1,94 @@
+import { Component, Element, Prop, State } from "@stencil/core";
+import { injectHistory, RouterHistory } from "@stencil/router";
+
+import CounterfactualTunnel from "../../data/counterfactual";
+import { AppInstance } from "../../data/mock-app-instance";
+import MockNodeProvider from "../../data/node-provider";
+import { cf, Node } from "../../data/types";
+
+declare var NodeProvider;
+declare var cf;
+
+@Component({
+  tag: "app-provider"
+})
+export class AppProvider {
+  @Element() private el: HTMLStencilElement = {} as HTMLStencilElement;
+  @Prop() history: RouterHistory = {} as RouterHistory;
+  @Prop() updateAppInstance: (appInstance: AppInstance) => void = () => {};
+  @Prop() updateAppFactory: (appFactory: cf.AppFactory) => void = () => {};
+  @Prop({ mutable: true })
+  nodeProvider: MockNodeProvider = {} as MockNodeProvider;
```

Comment:
why is this on a separate line?

Reply:
The line, all in one line, is 84 chars. So linting bumps it a line